ReactCommon/fabric/textlayoutmanager/platform/ios/RCTFontUtils.mm

```diff
@@ -137,8 +137,8 @@ static UIFontWeight RCTUIFontWeightFromFloat(CGFloat fontWeight) {
   RCTFontProperties defaultFontProperties = RCTDefaultFontProperties();
   fontProperties = RCTResolveFontProperties(fontProperties);
 
-  CGFloat effectiveFontSize =
-      fontProperties.sizeMultiplier * fontProperties.size;
+  CGFloat sizeMultiplier = !isnan(fontProperties.sizeMultiplier) ? fontProperties.sizeMultiplier : 1.0;
```

Comment:
That's not a proper fix of a problem.
We must have an `assert(!isnan(fontProperties.sizeMultiplier)` here, but not defaulting to `1.0`.

Instead, we have to figure out why it's happening (the default text attributes are probably wrong) and fix it there.

Reply:
Done. :)

Reply:
Thank you, Wu!!1